README.md

```diff
@@ -39,17 +40,20 @@ You can configure the behaviour and output of the `tldr` client by setting envir
     export TLDR_DOWNLOAD_CACHE_LOCATION="https://tldr-pages.github.io/assets/tldr.zip"
 
 ### Cache
+
+Cache is downloaded from `TLDR_DOWNLOAD_CACHE_LOCATION` (defaults to the one described in [the client specification](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#caching)), unzipped and extracted into the [local cache directory](#cache-location).
```

Comment:
Should probably mention that this only happens when you use the `--update` flag, otherwise it just grabs individual pages.